1209/ABCD/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: CodeBridge
+owner: Intelectron India
+license: OSHW
+site: https://www.intelectron.in
+source: https://github.com/sandeepansengupta/Generic/tree/master/CodeBridge/CAD/Eagle
```

Comment:
Please also include OSS-licensed source code for your product in your source repository.